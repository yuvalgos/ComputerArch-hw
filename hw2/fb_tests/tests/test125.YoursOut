L1miss=0.854 L2miss=0.600 AccTimeAvg=141.049
