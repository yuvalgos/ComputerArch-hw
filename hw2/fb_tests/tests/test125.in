r 0x77644
w 0x77658
r 0x77650
w 0x775f0
w 0x77600
r 0x77680
r 0x7761c
r 0x77654
r 0x77660
w 0x77664
w 0x77684
w 0x77614
r 0x77678
r 0x77678
r 0x77670
r 0x77614
w 0x77600
w 0x77644
w 0x77688
w 0x77608
r 0x77664
w 0x77624
r 0x77638
w 0x775e8
w 0x77604
w 0x775ec
r 0x775f0
w 0x775f8
r 0x77634
w 0x77658
r 0x77624
r 0x7767c
r 0x77688
r 0x7767c
r 0x77614
w 0x77688
r 0x7766c
w 0x775f0
w 0x77670
r 0x77650
r 0x77624
