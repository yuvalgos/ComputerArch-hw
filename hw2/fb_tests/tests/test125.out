L1miss=0.854 L2miss=0.629 AccTimeAvg=136.854
